### PR TITLE
[MPQEditor] Implement webviewPanel init.

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements webviewPanel initialization and update.

Its correctness is tested in #1511.

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1511
Previous draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: src/MPQEditor/MPQEditor.ts [hunk 1]
```diff
@@ -269,6 +271,75 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
     //TODO process messages
   }
 
+  /**
+   * @brief Initializes webviewpanel
+   */
```
Comment (seanshpark) [inline src/MPQEditor/MPQEditor.ts:276]: IMHO, this comment is needless. please remove.

**Review by dayo09 — APPROVED** [review]
LGTM for me :-D

File: src/MPQEditor/MPQEditor.ts (hunk 1 above)
Comment (stamalakhov) [reply src/MPQEditor/MPQEditor.ts]: Ok.
@seanshpark @dayo09 
done.

**Review by dayo09 — APPROVED** [review]

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**